test/core/componentGroupTests.ts

```diff
@@ -164,32 +126,54 @@ describe("ComponentGroups", () => {
+  describe("requests space based on contents, but occupies total offered space", () => {
+    var SVG_WIDTH = 400;
+    var SVG_HEIGHT = 400;
+
+    it("with no Components", () => {
+      var svg = generateSVG();
+      var cg = new Plottable.Component.Group([]);
+
+      var request = cg._requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      verifySpaceRequest(request, 0, 0, false, false, "empty Group doesn't request any space");
+
+      cg.renderTo(svg);
+      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
     });
 
-    it("works for a ComponentGroup with only proportional-size components", () => {
-      var cg = new Plottable.Component.Group();
-      var c1 = new Plottable.Component.AbstractComponent();
-      var c2 = new Plottable.Component.AbstractComponent();
-      cg.below(c1).below(c2);
-      var request = cg._requestedSpace(10, 10);
-      verifySpaceRequest(request, 0, 0, false, false, "");
+    it("with proportional Components", () => {
+      var svg = generateSVG();
+      var cg = new Plottable.Component.Group([
+        new Plottable.Component.AbstractComponent(),
+        new Plottable.Component.AbstractComponent()
+      ]);
+
+      var request = cg._requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      verifySpaceRequest(request, 0, 0, false, false, "empty Group doesn't request any space");
+
+      cg.renderTo(svg);
+      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
     });
 
-    it("works when there are fixed-size components", () => {
-      var cg = new Plottable.Component.Group();
-      var c1 = new Plottable.Component.AbstractComponent();
-      var c2 = new Plottable.Component.AbstractComponent();
-      var c3 = new Plottable.Component.AbstractComponent();
-      cg.below(c1).below(c2).below(c3);
-      fixComponentSize(c1, null, 10);
-      fixComponentSize(c2, null, 50);
-      var request = cg._requestedSpace(10, 10);
-      verifySpaceRequest(request, 0, 50, false, true, "");
+    it("with fixed-size Components", () => {
+      var svg = generateSVG();
+      var tall = new Mocks.FixedSizeComponent(SVG_WIDTH/4, SVG_WIDTH/2);
+      var wide = new Mocks.FixedSizeComponent(SVG_WIDTH/2, SVG_WIDTH/4);
+
+      var cg = new Plottable.Component.Group([tall, wide]);
+
+      var request = cg._requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      assert.strictEqual(request.width, SVG_WIDTH/2, "requested enough space for widest Component");
+      assert.strictEqual(request.height, SVG_HEIGHT/2, "requested enough space for tallest Component");
+
+      cg.renderTo(svg);
+      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
     });
```

Comment:
Not yet, since `DragBoxLayer` has yet to merge. There aren't many `Component`s that don't participate in layout but occupy all space in the final offer ... other than `Group` itself.